src/devices/Mcp25xxx/Register/MessageReceive/RxBxEid8.cs

```diff
@@ -0,0 +1,78 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+
+namespace Iot.Device.Mcp25xxx.Register.MessageReceive
+{
+    /// <summary>
+    /// Receive Buffer Extended Identifier High Register.
+    /// </summary>
+    public class RxBxEid8 : IRegister
```

Comment:
Some of these registers would work better with C-like bit fields - I wonder if you could achieve something similar with enum and static extension methods - this would simplify this code significantly I think (watch out with those since it's easy to start operating on copies when you use value types - perhaps the static extensions would have to be immutable and always return a copy)

Reply:
@krwq  Let me review this some more.  Until then, I'm sending commits on all the other changes.  Thx for feedback so far.

Reply:
@krwq what if we tried using a BitVector32?

Reply:
@krwq i just tried using a BitVector32 and didn't see any benefit over the current code.  Maybe if you had an example of the C bit fields/extension would help me understand a little better.

Reply:
perhaps something like this (not tested)

```csharp
public static class FooExtensions
{
   public static uint C(this Foo foo)
   {
     return (uint)foo >> 2;
   }

   public static void C(ref this Foo foo, uint value) // might be `this ref` or might not be possible - make sure this sets flag if you get rid of ref (you might be working on a copy)
   {
     foo |= (Foo)((value << 2) & 0b11);
   }
}
```

Reply:
I can only see 5 of the 35 registers where this might be useful.  I'd rather keep consistent across all for now.  Maybe we create a new issue to discuss and possibly would be helpful where bindings could be setup using in a common pattern.